gocourse05/camera/camera.go

```diff
@@ -0,0 +1,27 @@
+package camera
+
+import "time"
+
+const (
+	noLight = iota + 1
+	infraredLight
+	sunLight
+)
```

Comment:
Можна ж по красоті зробити за допомогою кастомного типу https://go.dev/doc/effective_go#constants:
```suggestion
type lightType int

const (
	noLight lightType = iota
	infraredLight
	sunLight
)
```

Також, `noLight == 0` гарно лягає на концепцію ["Make zero value useful"](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_make_the_zero_value_useful).